collector/consumer/processor/kindlingformatprocessor/inner_dictionary.go

```diff
@@ -237,3 +246,25 @@ func getSubStageStatus(requestSendTime int64) string {
 		return YellowStatus
 	}
 }
+
+func TraceValuesToLabel(cfg *Config, g *gauges) {
+	for i := 0; i < len(g.Values); i++ {
+		if g.Values[i].Name == constvalues.RequestSentTime {
+			g.targetLabels.AddIntValue(constlabels.RequestSentNs, g.Values[i].Value)
+		} else if g.Values[i].Name == constvalues.WaitingTtfbTime {
+			g.targetLabels.AddIntValue(constlabels.WaitingTTfbNs, g.Values[i].Value)
+		} else if g.Values[i].Name == constvalues.ContentDownloadTime {
+			g.targetLabels.AddIntValue(constlabels.ContentDownloadNs, g.Values[i].Value)
+		} else if g.Values[i].Name == constvalues.RequestTotalTime {
+			g.targetLabels.AddIntValue(constlabels.RequestTotalNs, g.Values[i].Value)
+		} else if g.Values[i].Name == constvalues.RequestIo {
+			g.targetLabels.AddIntValue(constlabels.RequestIoBytes, g.Values[i].Value)
+		} else if g.Values[i].Name == constvalues.ResponseIo {
+			g.targetLabels.AddIntValue(constlabels.ResponseIoBytes, g.Values[i].Value)
+		}
```

Comment:
Why not use a switch-case here? I think it will be more efficient.

Reply:
good idea